assets/data-port/import/data/actions.js

```diff
@@ -36,65 +41,51 @@ export const fetchFromAPI = ( request ) => ( {
 /**
  * Fetch importer data for current job.
  */
-export function* fetchCurrentJobState() {
-	yield startFetchCurrentJobState();
-
-	try {
+export const fetchCurrentJobState = composeFetchAction(
+	START_FETCH_CURRENT_JOB_STATE,
+	function*() {
 		const data = yield fetchFromAPI( {
-			path: buildJobEndpointUrl( API_SPECIAL_ACTIVE_JOB_ID ),
+			//path: buildJobEndpointUrl( API_SPECIAL_ACTIVE_JOB_ID ),
+			path: buildJobEndpointUrl( '3efc3b695a2a85c2ac635923cc604d4c' ),
```

Comment:
🔢 🔢 🔢 🔢

Reply:
Wow, how did THAT get in there? 😱 
Decreased number of hashes in 7e23991

Reply:
😆